ios/SalesforceReact/SFOauthReactBridge.m

```diff
@@ -77,12 +79,15 @@ - (void)sendAuthCredentials:(RCTResponseSenderBlock) callback
     if (nil != creds) {
         NSString *instanceUrl = creds.instanceUrl.absoluteString;
         NSString *loginUrl = [NSString stringWithFormat:@"%@://%@", creds.protocol, creds.domain];
+        NSString *communityUrl = creds.communityUrl ? creds.communityUrl.absoluteString : nil;
```

Comment:
However, you can't have nil values in a NSDictionary if you need to represent a null value in a dictionary, you should use NSNull (see https://developer.apple.com/documentation/foundation/nsdictionary?language=objc).

So instead of
```
kCommunityUrlCredentialsDictKey: communityUrl,
```
You should do
```
kCommunityUrlCredentialsDictKey: communityUrl ?: [NSNull null]
```

Reply:
@wmathurin Did the same for `communityId`, that's ok ?